### PR TITLE
Add command to remove knative.deployers for uninstall

### LINKED ISSUE #22: Uninstall of CRDs hang if there was a Knative Route defined
If I delete the helm install and run the command do remove the CRDs I can get into a situation where a `Route` can't be delete. Looks like there is a finalizer that hold things up:

```
    finalizers:
    - routes.serving.knative.dev
```

### PR DESCRIPTION
**Pull request by trisberg**
Fixes #22

### CODE REVIEW

**Comment by scothis** [comment]
@trisberg are you sure this fixes the issue and not that you happened to get lucky?

See https://github.com/knative/serving/issues/4732

**Comment by trisberg** [comment]
That looks like a different problem. I was deleting CRDs and not a namespace, and when I edited the orphaned Route resource removing the finalizer the deadlock disappeared and CRDs and Route got deleted.

File: README.md [hunk 1]
```diff
@@ -56,6 +56,9 @@ Helm charts to install Istio and riff.
 ### Uninstall
 
 ```
+# remove any Knative deployers (if Knative runtime is enabled)
+kubectl delete --all-namespaces deployer.knative.projectriff.io --all
```
Comment (scothis) [inline README.md]: Let's be more clear that all riff and knative (if installed) resource should be dropped

```suggestion
kubectl delete riff --all --all-namespaces
kubectl delete knative --all --all-namespaces
```

File: README.md [hunk 2]
```diff
@@ -56,13 +56,18 @@ Helm charts to install Istio and riff.
...
+
+# remove any Knative resources (if Knative runtime is enabled)
+kubectl delete knative --all-namespaces --all
+
 # remove riff
 helm delete --purge riff
 kubectl delete customresourcedefinitions.apiextensions.k8s.io -l app.kubernetes.io/managed-by=Tiller,app.kubernetes.io/instance=riff
 
 # remove istio (if installed)
 helm delete --purge istio
```
Comment (scothis) [inline README.md]:
```suggestion
helm delete --purge istio
kubectl get customresourcedefinitions.apiextensions.k8s.io -oname | grep istio.io | xargs -L1 kubectl delete
```

**Pull request merged**